angular/core/components/group_page/description_card/description_card.haml

```diff
@@ -0,0 +1,12 @@
+%section.description-card{aria-labelledby: "description-card-title"}
+  %h2#description-card-title.lmo-card-heading{translate: "group_page.description"}
+  %div{ng-hide: "editorEnabled"}
+    .description-card__placeholder.lmo-placeholder.lmo-hint-text{translate: "group_page.description_placeholder", ng-if: "showDescriptionPlaceholder()"}
```

Comment:
I reckon this could just be `ng-if: "!group.description"` to forgo the additional controller method